dateparser/conf.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+class Settings(object):
+    PREFER_DATES_FROM_FUTURE = None
```

Comment:
Perhaps it'd be better to change the setting into: PREFER_DATES_FROM
And give it three options: DEFAULT, PAST, FUTURE.